module-2/lab-mongo/mongo-main.ipynb

```diff
@@ -61,7 +61,7 @@
    "metadata": {},
```

Comment:
- "All the companies that have a null type of value on the category_code field". Null is not the same as None. I would query both separately. I encourage you to ask Marc the difference between Null and undefined in JavaScript. 
- Remember to ask Marc to solve question number 12!!